app.js

```diff
@@ -1,16 +1,16 @@
 let app = {};
 let ipcRenderer = require('electron').ipcRenderer;
-
+let $ = require('jquery');
```

Comment:
JQuery files were outdated. So i deleted them and used npm instead. Not sure if you can require in in line html script tag, but then whats the point? Jquery isnt used anywhere else.